src/main/java/org/growify/bank/service/strategy/UserService.java

```diff
@@ -0,0 +1,118 @@
+package org.growify.bank.service.strategy;
+
+import org.growify.bank.dto.request.UptadeRequestDTO;
+import org.growify.bank.dto.response.TokenResponseDTO;
+import org.growify.bank.dto.response.UserResponseDTO;
+import org.growify.bank.exception.TokenInvalidException;
+import org.growify.bank.model.token.Token;
+import org.growify.bank.model.token.TokenType;
+import org.growify.bank.model.user.User;
+import org.growify.bank.repository.TokenRepository;
+import org.growify.bank.repository.UserRepository;
+import org.modelmapper.ModelMapper;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import lombok.RequiredArgsConstructor;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+public class UserService {
+
+    private final TokenRepository tokenRepository;
+    private final UserRepository userRepository;
+    private final PasswordEncoder passwordEncoder;
+    private final AuthenticationTokenManagerStrategy authTokenManager;
+    private final AuthenticateValidationStrategy authValidationStrategy;
+    private final PasswordValidationStrategy passwordValidationStrategy;
+    private final UserIdValidationStrategy userIdValidationStrategy;
+    private final EmailAlreadyValidationStrategy emailAlreadyValidationStrategy;
+    private final ModelMapper modelMapper;
```

Comment:
There's no Model Mapper configured in the application context. A @Configuration class that publishes a `ModelMapper` bean is required to ensure dependency injection. An issue must be created to proceed.

Reply:
Great observation. In fact, the ModelMapper wasn’t configured yet during the development of the "UserService.java" class. I’ll make sure it’s properly injected.